### PR TITLE
Improved image pane

### PR DESCRIPTION
**Pull request by dodumosu**
Implements the changes requested on nditech/apollo-issues#180

### CODE REVIEW

**Comment by dodumosu** [comment]
@takinbo would we still need the image gallery with this feature? it's been left in for now, though

**Comment by takinbo** [comment]
Answer is yes we would still need the image gallery. Any progress on the issue regarding the icon display on Chromium?

**Comment by dodumosu** [comment]
> Answer is yes we would still need the image gallery. Any progress on the issue regarding the icon display on Chromium?

none. it happens in Google Chrome as well. the author of the component recommended changing the browser version. i still have a card to play, though

**Comment by takinbo** [comment]
@dodumosu Before I merge this, could you also change the behaviour of the thumbnails so that when they are clicked on, they load the image in the split view rather than in the Lightbox as it currently does?

**Comment by dodumosu** [comment]
> @dodumosu Before I merge this, could you also change the behaviour of the thumbnails so that when they are clicked on, they load the image in the split view rather than in the Lightbox as it currently does?

done. waiting for feedback

File: apollo/frontend/templates/frontend/submission_edit.html [hunk 1]
```diff
@@ -383,12 +382,12 @@ <h5 class="mb-0">{{ group.name }}</h5>
...
       </div>
-      <div class="card-body">
+      <div id="carousel-body-el" class="card-body">
         <div class='slick m-3'>
           {% for item in submission.get_image_data() -%}
           {% if item['attachment'] %}
-          <div class="d-flex flex-column align-items-center justify-content-center mb-3" data-gallery-item-tag="{{ item['tag'] }}">
-            <a href="{{ item['attachment'].photo.url }}" data-lightbox="images" data-title="{{ item['tag'] }} &middot; {{ item['description'] }}">
+          <div class="d-flex flex-column align-items-center justify-content-center mb-3" data-gallery-item-tag="{{ item['tag'] }}" style="cursor: pointer;">
+            <a data-lightbox="images" data-title="{{ item['tag'] }} &middot; {{ item['description'] }}">
```
Comment (takinbo) [inline apollo/frontend/templates/frontend/submission_edit.html]: Could you also remove the extraneous references to Lightbox in the markup?

**Review by takinbo — CHANGES_REQUESTED** [review]

**Review by takinbo — APPROVED** [review]
LGTM

**Pull request merged**